### DISCUSSION TITLE
[주간 회고] 프로젝트 2주차 (2024.07.26)

### DISCUSSION THREAD

**Discussion by eun-byeol** [woowacourse-teams/2024-ody#310] [General]
7월 26일 금요일 5:30 ~ 6:00까지 KPT 주간 회고를 진행했습니다.
> Keep - Problem - Try 적용하기
    - Keep : 잘하고 있는 점. 계속 했으면 좋겠다 싶은 점.
    - Problem : 뭔가 문제가 있다 싶은 점. 변화가 필요한 점.
    - Try : 잘하고 있는 것을 더 잘하기 위해서, 문제가 있는 점을 해결하기 위해서 우리가 시도해 볼 것들

# Keep
![image](https://github.com/user-attachments/assets/ab57abc9-9e5d-4174-96d4-53729b76426e)

# Problem
![image](https://github.com/user-attachments/assets/745546f1-3e16-4671-b1d5-82c1734d6f4d)

# Try
- 개발 시작 전 미리  BE - AN 간 연결 테스트 해보기
- 코드 리뷰를 파트 별로 컨벤션 회의를 통해 개선해보자.
- 상황 공유를 확실히 하고, 전달 받은 사람이 그 상황을 이해했다면 Sign을 주자. 📝
- 스크럼 때 현재 **상황** 공유를 좀 더 전달해줬으면 좋겠다.
    - ex) 어떠한 이슈를 어떠한 문제로 헤매고 있음. 언제언제 PR 날릴 것 같음.
- 빼먹는 기능이 있을 수 있으니 미리 이슈 다 파는 것도 좋을듯 ?
- 확인 이모지 잘 달기

**Comment by aprilgom** [comment]
수다날 좋아요~
